### PR TITLE
[1LP][RFR] Adding services requester drop down sorting test

### PR DESCRIPTION
**Pull request by niyazRedhat**
### PRT Run
{{pytest: cfme/tests/services/test_request.py::test_services_requester_dropdown_sorting -svv --long-running }}

- automated services requester drop down sorting test
- Removed deprioritized low/non required tests
- Changed caseimportance for 3 tests

### CODE REVIEW

File: cfme/services/requests.py [hunk 1]
```diff
@@ -284,6 +285,19 @@ def in_requests(self):
...
+
+        apply = Button('Apply')
+        reset = Button('Reset')
+
+        requester = BootstrapSelect(
+            locator=(
+                './/div[contains(@class, "requester")]'
+                '/div[contains(@class, "bootstrap-select")]'
+            )
+        )
```
Comment (valaparthvi) [inline cfme/services/requests.py:299]:
```suggestion
        requester = BootstrapSelect(view, name="user_choice")
```

File: cfme/tests/services/test_request.py [hunk 2]
```diff
@@ -39,10 +40,9 @@ def test_copy_request_bz1194479(appliance, provider, catalog_item, request):
     assert navigate_to(service_request, 'Details')
 
 
-@pytest.mark.meta(coverage=[1749953])
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_services_requester_dropdown_sorting():
+@pytest.mark.meta(automates=[1749953])
```
Comment (valaparthvi) [inline cfme/tests/services/test_request.py:43]: Can you add `1767660` to the list?

File: cfme/tests/services/test_request.py [hunk 3]
```diff
@@ -3,8 +3,10 @@
 from cfme import test_requirements
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.services.requests import RequestsView
 from cfme.services.service_catalogs import ServiceCatalogs
 from cfme.utils.appliance.implementations.ui import navigate_to
+from widgetastic_manageiq import BootstrapSelect
```
Comment (valaparthvi) [inline cfme/tests/services/test_request.py:9]: It's better if you import `BootstrapSelect` from `widgetastic_patternfly`.

File: cfme/tests/services/test_request.py [hunk 4]
```diff
@@ -63,4 +65,11 @@ def test_services_requester_dropdown_sorting():
             3.
             4. Requester dropdown should Be Organized alphabetically
     """
-    pass
+    request = ServiceCatalogs(
+        appliance, catalog=generic_catalog_item.catalog, name=generic_catalog_item.name
+    ).order()
+    view = request.create_view(RequestsView)
+    requester = BootstrapSelect(view, name="user_choice")
```
Comment (digitronik) [inline cfme/tests/services/test_request.py:72]: Required:
I think you can add this widget in `RequestView`

**Pull request merged**